toolkit/scss/shared_placeholders/_temporary-messages.scss

```diff
@@ -6,4 +6,13 @@
   background: $govuk-blue;
   color: $white;
   padding: $gutter-half $gutter-half 18px $gutter-half;
+
+  a, a:visited {
+    color: $white;
+  }
+
+  a:hover, a:active {
+    @extend a;
+    text-decoration: none;
```

Comment:
It's your call - underline going might look nicer, but not sure if consistency across the site trumps that?